utilities/state_converters.meta

```diff
@@ -187,7 +187,7 @@
   dimensions = (horizontal_loop_extent, vertical_layer_dimension)
   intent = in
 [ rho ]
-   standard_name = density_of_dry_air_at_stp
+   standard_name = density_of_dry_air
```

Comment:
Following the Standard Names dictionary (https://github.com/ESCOMP/CCPPStandardNames/blob/main/Metadata-standard-names.md), this should probably be: `dry_air_density`

Reply:
Name changed.